docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -2,7 +2,44 @@
 [[ref-daemon-config-files-provisiond]]
 = Provisiond
 
-Handles scanning of requisition data to xref:operation:provisioning/introduction.adoc[provision] into monitored nodes.
+The provision daemon handles scanning of requisition data to xref:operation:provisioning/introduction.adoc[provision] into monitored nodes.
```

Comment:
This could be a case of "provision" meaning too many things, but I don't think this is quite the process. It sounds like the provisioning is happening into the monitored nodes, whereas I think of provisioning as identifying the nodes you want to monitor in Horizon ("importing" the nodes).

Not sure what to suggest on Friday afternoon. :-)

Reply:
@mmahacek Would you have a suggestion about how to clarify this sentence? :)

Reply:
Provisiond scans nodes a) from requisition/import files and b) from newSuspect events to import into the database. Only nodes in the database are monitored by other daemons. 

NewSuspect events are triggered by discover scans and by trapd/syslogd if those daemons are configured to create the event if an incoming message doesn't match an existing node.

Reply:
Thanks!

```suggestion
The provision daemon scans nodes from requisition and import files, and from newSuspect events.
The scanned nodes are then added to the database.
Note that only nodes in the database are monitored by other daemons.

NewSuspect events are triggered by Discover scans, and by trapd and syslogd if those daemons are configured to create the event.
When configured properly, trapd and syslogd create a newSuspect event if an incoming message does not match any existing nodes.
```